### PR TITLE
Theme button

### PR DESCRIPTION
**Pull request by DenysShurubura**
https://www.loom.com/share/8b5e14517e494b4ab1272ed68dfdc9f5?sid=3f6a1109-fa9a-442d-86f7-b08b93264061

### CODE REVIEW

File: src/modules/shared/Header/Header.tsx [hunk 1]
```diff
@@ -116,17 +129,17 @@ export const Header: React.FC = () => {
           onClick={() => setIsBurgerActive(!isBurgerActive)}
         >
           {isBurgerActive ? (
-            <Close color={isThemeDark ? '#fff' : '#000'} />
+            <Close color={isDarkTheme ? '#fff' : '#000'} />
```
Comment (mpohorenyi) [inline src/modules/shared/Header/Header.tsx]: For bright theme: #0F0F11
For dark theme: #F1F2F9

File: src/modules/shared/Header/Header.tsx [hunk 2]
```diff
@@ -116,17 +129,17 @@ export const Header: React.FC = () => {
...
           ) : (
-            <Burger color={isThemeDark ? '#fff' : '#000'} />
+            <Burger color={isDarkTheme ? '#fff' : '#000'} />
```
Comment (mpohorenyi) [inline src/modules/shared/Header/Header.tsx]: Create a variable for this. Place it outside the body of the component

File: src/modules/shared/Header/Header.tsx [hunk 3]
```diff
@@ -23,24 +23,22 @@ const getLinkClass = (
...
 export const Header: React.FC = () => {
-  // const { isDarkTheme } = useAppSelector(state => state.theme);
-  // const dispatch = useAppDispatch();
+  const { isDarkTheme } = useAppSelector(state => state.theme);
+  const dispatch = useAppDispatch();
 
-  // function ThemeHandler() {
-  //   dispatch(actions.change());
-  // }
+  function ThemeHandler() {
```
Comment (mpohorenyi) [inline src/modules/shared/Header/Header.tsx]:
```suggestion
  function themeHandler() {

File: src/modules/shared/Header/Header.tsx [hunk 4]
```diff
@@ -49,7 +47,7 @@ export const Header: React.FC = () => {
           onClick={() => setIsBurgerActive(false)}
         >
           <img
-            src={!isThemeDark ? logo : logoDark}
+            src={!isDarkTheme ? logo : logoDark}
```
Comment (mpohorenyi) [inline src/modules/shared/Header/Header.tsx:56]: Use .svg as a react component

**Review by mpohorenyi — CHANGES_REQUESTED** [review]
awesome! 🔥

Please correct minor comments

File: src/modules/shared/Header/Header.tsx [hunk 5]
```diff
@@ -92,10 +90,25 @@ export const Header: React.FC = () => {
       </div>
 
       <div className={styles.header__right}>
+        <div className={styles.header__themeSwitch}>
+          <input
+            type="checkbox"
+            className={styles.header__themeSwitch_input}
+            id="themeSwitch"
+            onChange={() => ThemeHandler()}
+          />
```
Comment (OlhaKoziuk) [inline src/modules/shared/Header/Header.tsx:105]: its ok, but you can reduce onChange={ThemeHandler} and why with a capital letter? it`s not a function-constructor

**Review by OlhaKoziuk — APPROVED** [review]

**Review by mpohorenyi — APPROVED** [review]
Great Job! 🔥

**Pull request merged**